homeassistant/components/generic/config_flow.py

```diff
@@ -426,24 +426,12 @@ async def async_step_init(
                     # is always jpeg
                     still_format = "image/jpeg"
                 data = {
-                    CONF_AUTHENTICATION: user_input.get(CONF_AUTHENTICATION),
-                    CONF_STREAM_SOURCE: user_input.get(CONF_STREAM_SOURCE),
-                    CONF_PASSWORD: user_input.get(CONF_PASSWORD),
-                    CONF_STILL_IMAGE_URL: user_input.get(CONF_STILL_IMAGE_URL),
+                    CONF_USE_WALLCLOCK_AS_TIMESTAMPS: self.config_entry.options.get(
+                        CONF_USE_WALLCLOCK_AS_TIMESTAMPS, False
+                    ),
+                    **user_input,
                     CONF_CONTENT_TYPE: still_format
                     or self.config_entry.options.get(CONF_CONTENT_TYPE),
```

Comment:
This could never be anything than `"image/jpeg"` according to the above logic?

Reply:
If still_url is defined, then the image type could be something other than image/jpeg.

Reply:
My bad. I read `still_format` and `still_url` as same.